packages/accounts-base/accounts_server.js

```diff
@@ -1,10 +1,36 @@
 var crypto = Npm.require('crypto');
 
+// Constructor for the Accounts namespace on the server. Note that this
+// constructor is less likely to be instantiated multiple times than the
+// AccountsClient constructor, because a single server can provide only
+// one set of methods.
+AccountsServer = function AccountsServer(ddpUrl) {
+  AccountsCommon.call(this, ddpUrl);
+
+  // list of all registered handlers.
+  this._loginHandlers = [];
+
+  setupUsersCollection(this._users);
+  setupDefaultLoginHandlers(this);
+  setExpireTokensInterval(this);
+
+  this._validateNewUserHooks = [
+    defaultValidateNewUserHook
+  ];
+
+  this._deleteSavedTokensForAllUsersOnStartup();
```

Comment:
Here's an example of some code that used to run at the top level getting called from a constructor function, so that it will run for each accounts instance that might be created.